conv/abci/block.go

```diff
@@ -98,6 +98,21 @@ func ToABCIBlock(block *types.Block) (*tmtypes.Block, error) {
 	return &abciBlock, nil
 }
 
+func ToABCIBlockMeta(block *types.Block) (*tmtypes.BlockMeta, error) {
```

Comment:
`BlockMeta` afaik isn't used much in the codebase. Might make sense to add this as utility function in the client itself ?